### ISSUE TITLE
Trailing comma for function with one arg

### ISSUE THREAD

**Issue by cosinequanon** [psf/black#880]
A function with one argument keeps the trailing comma but if there are two it is stripped.

In:
```
def one(
    a,
):
    pass

def two(
    a,
    b,
):
    pass
```
Out
```
def one(a,):
    pass


def two(a, b):
    pass
```
From https://black.now.sh/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4ACGAFNdAD2IimZxl1N_WlbvK5V-4TOOTwYyJEleH-JCdis69M0BXQy1O1PWCxUpRscYAufHo8BjrjIj1dM6YL_cyWPemFXxt1cpBhl0f8lWpI3sZ5u99cYAAACFuqAGuJSBnwABb4cBAAAAtWKNgbHEZ_sCAAAAAARZWg==

**Comment by zsol** [comment]
That's a bug, `one` should be formatted like so:
```
def one(a):
    pass
```

**Comment by drunkwcodes** [comment]
It is still runnable.

**Comment by stagedatalosstest** [comment]
test123

**Comment by dylanjblack** [comment]
I made pull request #891 with a potential fix. Please let me know if the approach makes sense and if you have any suggestions for improvement.

**Comment by Jma353** [comment]
Looks like this was fixed in 9394de150ebf0adc426523f46dc08e8b2b2b0b63.  Can this be closed?

**Comment by zsol** [comment]
yep.
